lib/api/context/SdmContext.ts

```diff
@@ -48,6 +49,10 @@ export interface SdmContext {
      */
     credentials: ProjectOperationCredentials;
 
+    /**
+     *
```

Comment:
Did you mean to put some sort of comment in here?

Reply:
I think i should, shouldn't I?